coalib/output/ConsoleInteraction.py

```diff
@@ -322,12 +323,20 @@ def print_results_formatted(log_printer,
 
             for range in result.affected_code:
                 format_args['affected_code'] = range
-                print(format_str.format(file=range.start.file,
-                                        line=range.start.line,
-                                        end_line=range.end.line,
+                file_name = range.start.file
+                start_line = range.start.line
+                end_line = range.end.line
+                source_lines = result.get_affected_source_lines(file_name,
+                                                                start_line,
+                                                                end_line)
```

Comment:
can't you please add this to the `format_args` dict? That's what it's for. Then you don't need to give the args explicitly below